demo/kserve/scripts/install/kserve-install.sh

```diff
@@ -54,31 +61,40 @@ then
     exit 1
 fi
 
-if [[ ! -n ${TARGET_OPERATOR+x} ]]
+if [[ ! -n "${DEPLOY_OPERATOR+x}" ]]
```

Comment:
What is this variable for? 
What you want is to check the cluster if the kserve is installed or not and if so, do you want to skip installation?
Why don't we add the checking logic as well?  Then `DEPLOY_OPERATOR` can be changed to `FORCE_DEPLOY_OPERATOR` then. it means that by default, the script will not go through all steps about odh operator/kserve installation when those are already installed in the cluster but if `FORCE_DEPLOY_OPERATOR` set to `yes`, then it will try to install it. WDYT?

Reply:
>What you want is to check the cluster if the kserve is installed or not and if so, do you want to skip installation?


no it's not what I wanted to do. I just want to skip make RHODS installation optional. If DEPLOY_OPERATOR is `true` then RHODS/ODH gets installed. The generic name is based on the other variable names format (e.g., TARGET_OPERATOR)

> FORCE_DEPLOY_OPERATOR
Not sure it's much different from what is implemented in this PR. The differences, correct me if I'm wrong, would be:
1. by default RHODS/ODH doesn't get installed
2. logic to check if RHODS is already installed -> however, why adding this logic if we just install the operator regardless by setting the FORCE_DEPLOY_OPERATOR?

Reply:
it won't update but it will create a new sub, a new catalogsource and a new operator group if you used different names (if you use olminstall script, it will happens for sure). I believe it is better to avoid installing it at all. Also for a developer using the script may feel more safe if it can control RHODS installation. 

Anyways, feel free to close the PR if you think it's not necessary, not a problem